api/src/main/resources/hibernate.cfg.xml

```diff
@@ -32,7 +32,7 @@
 		<mapping resource="org/openmrs/api/db/hibernate/ConceptNameTag.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/ConceptClass.hbm.xml" />
 		<mapping resource="org/openmrs/api/db/hibernate/ConceptDatatype.hbm.xml" />
-		<mapping resource="org/openmrs/api/db/hibernate/ConceptProposal.hbm.xml" />
+		<mapping class="org.openmrs.ConceptProposal"/>
```

Comment:
This isn't how we've handled any other of the migrations. Please look at similar tickets for how this is done.